registry/client.go

```diff
@@ -352,3 +352,118 @@ func (e Error) Error() string {
 
 	return "registry error: " + strconv.Itoa(e.StatusCode)
 }
+
+// Compatibility levels.
+const (
+	BackwardCL           string = "BACKWARD"
+	BackwardTransitiveCL string = "BACKWARD_TRANSITIVE"
+	ForwardCL            string = "FORWARD"
+	ForwardTransitiveCL  string = "FORWARD_TRANSITIVE"
+	FullCL               string = "FULL"
+	FullTransitiveCL     string = "FULL_TRANSITIVE"
+	NoneCL               string = "NONE"
+)
+
+/*
+ValidCompatibilityLevel returns whether or not a compatibility level
+is valid according to the ones described in:
+https://docs.confluent.io/platform/current/schema-registry/develop/api.html#compatibility
+.
+*/
+func ValidCompatibilityLevel(compatibilityLevel string) bool {
```

Comment:
What is the reason to make this public? You already expose the levels.

Reply:
Exclusively to call it in the unit tests

Reply:
dumbness of mine, I moved it to client_internal_test.go

Reply:
There is not need to expose it just for the test as it muddies the API of the package. It should be tested via the error tests in the functions themselves, like `TestClient_SetGlobalCompatibilityLevelHandlesInvalidLevel`.

Reply:
Did my best to put also these tests